inference-engine/src/transformations/src/transformations/op_conversions/convert_sequences_to_tensor_iterator.cpp

```diff
@@ -93,12 +81,12 @@ ngraph::pass::ConvertRNNSequenceToTensorIterator::ConvertRNNSequenceToTensorIter
                                                                     pattern::any_input(),
                                                                     pattern::any_input(),
                                                                     pattern::any_input()});
-    ngraph::matcher_pass_callback callback = [](ngraph::pattern::Matcher &m) {
+    ngraph::matcher_pass_callback callback = [this](ngraph::pattern::Matcher &m) {
         auto sequence = std::dynamic_pointer_cast<ngraph::opset5::RNNSequence>(m.get_match_root());
 
         // Bidirectional Sequence op should be decomposed to Reverse + Forward
         // (e.g. apply BidirectionalRNNSequenceDecomposition transformation before this one)
-        if (!sequence || sequence->get_direction() == ngraph::op::RecurrentSequenceDirection::BIDIRECTIONAL) {
+        if (!sequence || sequence->get_direction() == ngraph::op::RecurrentSequenceDirection::BIDIRECTIONAL || transformation_callback(sequence)) {
```

Comment:
Maybe we can pass `transformation_callback` as a parameter instead of `this`?

Reply:
What do you mean? transformation_callback is a class method, it's not static. How I can do it?

Reply:
The usage of transformation callback is correct here.